tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
I think we are still going to have a problem going forward.  If @lsm5 updates to use container-storage-setup inside of docker.service and it specifies correctly CONTAINER_THINPOOL=container-pool, then on upgrade this is going to cause us issues.